### PR TITLE
Clarify grading view #37

### PR DESCRIPTION
**Pull request by nlhkh**

### CODE REVIEW

**Comment by nlhkh** [comment]
@tsauvine I have compared the `schema.rb` file from the master branch and this branch to make sure that the migrations produce the same schema, including column names and column types.

File: app/views/reviews/_review_editor.html.erb [hunk 1]
```diff
@@ -172,14 +172,16 @@
           <span data-bind="text: averageGrade"></span>
         </h3>
       <!-- /ko -->
-      
+
       <!-- Next page button -->
-      <!-- ko if: nextPage || $root.finishable() -->
-        <button class="btn" data-bind="click: $root.showNextPage">
+        <button class="btn" data-bind="enable: nextPage || $root.finishable(), click: $root.showNextPage">
```
Comment (tsauvine) [inline app/views/reviews/_review_editor.html.erb]: I'm confused about the comparison view. For example these are marked with a +:
+    t.string   "positive_caption"
+    t.string   "negative_caption"
+    t.string   "neutral_caption"

Apparently they are there in master, so are these additions or deletions? They are deprecated columns, nevertheless.

Comment (nlhkh) [inline app/views/reviews/_review_editor.html.erb]: They are additions. I found them in master. I shall remove them then. Do you remember other deprecated columns?

Comment (nlhkh) [inline app/views/reviews/_review_editor.html.erb]: @tsauvine I updated this pull request. You can review it and give me the signal to merge.

**Pull request merged**